generators/entity-server/templates/src/main/java/package/domain/Entity.java.jhi.ejs

```diff
@@ -0,0 +1,371 @@
+<%#
+ Copyright 2013-2021 the original author or authors from the JHipster project.
+
+ This file is part of the JHipster project, see https://www.jhipster.tech/
+ for more information.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      https://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-%>
+<&_
+ // Register sections 0 for unlimited childs, 1 for only one child.
```

Comment:
```suggestion
 // Register sections 0 for unlimited children, 1 for only one child.
```

0 and 1... I thought we weren't coding in C anymore :trollface: 
All joking aside, this clearly should be improved, to use booleans or something else for clarity.

I consider writing comments in code an anti-pattern as it shows the code isn't clear enough.

Reply:
Api is not accessible, so it’s better to add documentation here.
Or provide a better name. I can’t think in any.

Reply:
That's my point: there shouldn't be any.
The code should be clear enough so as not to need any doc (code as documentation).
The API, even though it's private, should be better: there shouldn't be any 0 or 1, maybe booleans, or whatever.